### PR TITLE
(fix) fix default envrc syntax

### PR DESCRIPTION
**Pull request by symphorien**
- [ ] Amended the changelog in `release.nix` (see `release.nix` for instructions)

### CODE REVIEW

File: src/default-envrc [hunk 1]
```diff
@@ -1,6 +1,9 @@
+#!/usr/bin/env bash
+# the shebang is ignored, but nice for editors
+
 if type -P lorri &>/dev/null; then
-  eval \"$(lorri direnv)\"
+  eval "$(lorri direnv)"
```
Comment (Profpatsch) [inline src/default-envrc:5]: I *think* these escapes need to be there? not sure

File: src/default-envrc [hunk 2]
```diff
@@ -1,6 +1,9 @@
+#!/usr/bin/env bash
+# the shebang is ignored, but nice for editors
```
Comment (Profpatsch) [inline src/default-envrc:2]: I don’t see how it could hurt, so +1

File: src/default-envrc (hunk 1 above)
Comment (symphorien) [reply src/default-envrc:5]: no they break the envrc:
```
direnv: loading ~/src/petgraph/.envrc                                                                                                                                                        
./.envrc: eval:5: unexpected EOF while looking for matching ``'
./.envrc: eval:6: syntax error: unexpected end of file
```

Comment (Profpatsch) [reply src/default-envrc:5]: Strange, that hasn’t happened to me before.

However:

```bash
$ help eval
eval: eval [arg ...]
    Execute arguments as a shell command.

    Combine ARGs into a single string, use the result as input to the shell,
    and execute the resulting commands.
```

So I guess the args get concatenated anyway by eval.

**Pull request merged**